input/europe/eu_very_crude.csv

```diff
@@ -1,104 +0,0 @@
-belgium,Doel1,454,88,29
```

Comment:
It has been removed. I thought having two files with similar data was confusing, so I left the PRIS data ( the more reliable one) and deleted the one from Wikidata